src/App.js

```diff
@@ -11,35 +11,35 @@ import '@knapsack-cloud/msk-design-system/dist/main.css';
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 
 function App() {
-  let footerLinks=[
+  let footerLinks = [
     { text: 'Example', href: 'https://example.com' },
     { text: 'Hello, world', href: 'https://helloworld.com' },
     { text: 'FooBar', href: 'https://foobar.com' },
   ];
   return (
-    <MskSiteWrapper>
+    <div>
```

Comment:
Should this wrapper div be changed to a JSX fragment? aka `<></>`? Just want to reduce as much extraneous markup as possible to have the best test of the components.